### ISSUE TITLE
A menu in a modal form does not open with shortcut key

### ISSUE THREAD

**Issue by beru** [beru/nana#14]

**Comment by beru** [comment]
Below are the sample code to create a modal dialog with a menu.

``` c++
#include <nana/gui.hpp>
#include <nana/gui/widgets/menubar.hpp>
#include <nana/gui/widgets/textbox.hpp>

using namespace nana;

int main()
{
    using namespace nana;
    form fm(API::make_center(400, 500));
    textbox txt1(fm, rectangle(10, 60, 200, 30));
    textbox txt2(fm, rectangle(10, 110, 200, 30));

    menubar mb;
    mb.create(fm);
    mb.push_back(L"&Hurr");
    mb.push_back(L"&durr");
    mb.at(0).append(L"&I'ma", [&](menu::item_proxy& ip){
        //API::focus_window(txt1);
    });
    mb.at(0).append(L"&sheep", [&](menu::item_proxy& ip){
        //API::focus_window(txt2);
    });
    mb.at(1).append(L"&1", [](menu::item_proxy& ip){

    });
    mb.at(1).append(L"&2", [](menu::item_proxy& ip){

    });
    mb.at(1).enabled(0, false);
    //mb.at(1).enabled(1, false);

#if 1
    fm.modality();
#else
    fm.show();
    exec();
#endif
}
```

**Comment by beru** [comment]
the fix has been merged into cnjinhao/nana/hotfixes-1.0.1 branch.
